src/_pytest/_code/source.py

```diff
@@ -60,7 +60,7 @@ def __getitem__(self, key: int) -> str:
         raise NotImplementedError()
 
     @overload  # noqa: F811
-    def __getitem__(self, key: slice) -> "Source":
+    def __getitem__(self, key: slice) -> "Source":  # noqa: F811
```

Comment:
There seems to be a discrepancy between where CI flake8 and my flake8 report this - on the decorator or the `def`. Maybe because of Python 3.8? So for now I add it to both.

Reply:
> Maybe because of Python 3.8?

Very likely, yes.
Isn't there a flake8 plugin to handle this automatically maybe?

Reply:
There seems to be some discussion regarding this [here](https://github.com/PyCQA/pyflakes/issues/488), I'm too lazy to read through it though...